scalahost/sbt/src/main/scala/scala/meta/scalahost/sbt/ScalahostSbtPlugin.scala

```diff
@@ -60,10 +60,7 @@ object ScalahostSbtPlugin extends AutoPlugin {
     javaOptions ++= {
       if (scalametaDependencies.value.isEmpty) Nil
       else {
-        val sourcepath =
-          scalahostSourcepath.value
-            .flatMap(_.map(_.getAbsolutePath))
-            .mkString(java.io.File.pathSeparator)
+        val sourcepath = sys.props("user.dir")
```

Comment:
We may hit on performance issues in `Fragment.deep/find` with this sourcepath. For example, in the scalafmt/scalameta repos `target/` often contains (tens of) thousands of files from testkit.